test/metabase/api/geojson_test.clj

```diff
@@ -140,7 +143,23 @@
              (mt/user-http-request :crowberto :get 200 "geojson" :url test-geojson-url))))
     (testing "error is returned if URL connection fails"
       (is (= "GeoJSON URL failed to load"
-             (mt/user-http-request :crowberto :get 400 "geojson" :url test-broken-geojson-url))))
+             (mt/user-http-request :crowberto :get 400 "geojson"
+                                   :url test-broken-geojson-url))))
+    (testing "error is returned if URL server never responds (#28752)"
+      (let [never-responder    (fn silent-async-handler [_request _respond _raise])
+            server             (ring-jetty/run-jetty never-responder
+                                                     {:join?         false
+                                                      :async?        true
+                                                      :port          0
+                                                      :async-timeout 60000})
+            port               (.. server getURI getPort)
+            never-responds-url (str "http://localhost:" port)]
+        (try
```

Comment:
could we have a comment on why a standalone server is needed here?

Reply:
added. good call